data/transactions/logic/evalAppTxn_test.go

```diff
@@ -1760,8 +1798,8 @@ itxn_submit
 int 1
 `
 
-	for _, unified := range []bool{true, false} {
-		t.Run(fmt.Sprintf("unified=%t", unified), func(t *testing.T) { //nolint:paralleltest // NO t.Parallel(). unified variable is actually shared
+	for _, unified := range []bool{true, false} { //nolint:paralleltest // NO t.Parallel(). unified variable is actually shared
```

Comment:
@jdtzmn I realize the comment predates the PR.  In my inspection, I did _not_ see a shared usage here.  Did you catch _why_ the test can't be marked as `t.Parallel`?

I don't think marking it as `t.Parallel` provides a meaningful performance improvement.  I raise the question because I think it reduces cognitive load to minimize parallel test exceptions.

If we agree it looks reasonable to change, I supplied a child PR:  https://github.com/jdtzmn/go-algorand/pull/5.

Reply:
I can also repoint the child PR against `master`.  Particularly because the PR makes _many_ changes (susceptible to merge conflict), I'm not looking to hold up merging the PR for the thread's question.

Reply:
Resolving - I repointed the PR via https://github.com/algorand/go-algorand/pull/4937.